content/terms/explanation/_index.md

```diff
@@ -0,0 +1,4 @@
+---
+title: Explanation
```

Comment:
We have a plural consistency issue: “Tutorials” and “Guidelines” is plural, but “Explanation” and “How to” is singular.
I would be in favor of having every section title in plural.

Reply:
I personally find it strange that these are singular and would prefer to have everything plural.

Reply:
After IRL discussion, let's go with all plural